dist/v2.2.0/account-info-nz-swagger.yaml

```diff
@@ -0,0 +1,4446 @@
+swagger: "2.0"
+info:
+  title: Account and Transaction API Specification
+  description: >-
+    Swagger for Account and Transaction API Specification. This is heavily
+    derived from the Open Banking UK API - see www.openbanking.org.uk for
+    details.
+  termsOfService: "https://www.apicentre.paymentsnz.co.nz/contact-us/"
+  contact:
+    name: Payments NZ API Centre
+    email: apicentre@paymentsnz.co.nz
+  license:
+    name: Licence
+    url: "https://www.apicentre.paymentsnz.co.nz/join/"
+  version: v2.1.0
+basePath: /open-banking-nz/v2.1
```

Comment:
Suspected typo for lines 15 and 16. Version and version in the path should be 2.2.0?